.github/workflows/ci_any_pr.yaml

```diff
@@ -86,10 +86,8 @@ jobs:
       - name: Golang unit tests
         timeout-minutes: 20
         env:
-          # these env variables are required by the golang tests
-          GOOGLE_CLOUD_PROJECT: ${{ secrets.GOOGLE_CLOUD_PROJECT }}
-          GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           GOLANG_TEST_DIR: ${{ matrix.go-test-dir }}
         run: |
+          source /var/local/gh-runner/env_vars
```

Comment:
Was this file  *manually* created (in the GitHub action runner virtual machines)?
If so, do you think we need to update the documentation here: https://github.com/GoogleCloudPlatform/anthos-samples/blob/806db81be21edd48a40149aded6127a2ecc15c01/.github/gh_runner.md?

Reply:
I also noticed that the value of `GOOGLE_APPLICATION_CREDENTIALS` (in the `/var/local/gh-runner/env_vars` file) in both virtual machines (GitHub Action runners) is pointing to a file. Does this file need to be manually created when creating a new GitHub Action? If so, should we mention that in https://github.com/GoogleCloudPlatform/anthos-samples/blob/806db81be21edd48a40149aded6127a2ecc15c01/.github/gh_runner.md?

Reply:
@NimJay - Yes the files were manually created this time. How they would be created when a new runner is added is by executing [these instructions](https://github.com/GoogleCloudPlatform/anthos-samples/blob/806db81be21edd48a40149aded6127a2ecc15c01/.github/gh_runner_dependencies.sh#L44-L53) **printed** at the end of the `gh_runner_dependencies.sh` script. I'll update the doc to add a note about completing the steps printed at the end of the script.

Yes, it points to the `Service Account` key file from the project. Creation of this file in a new runner is also part of the instructions at the end of the  `gh_runner_dependencies.sh` script

Reply:
Ohhh, I didn't realize that the instructions contain a step about `gh_runner_dependencies.sh`!
Nonetheless, the new message is perfect!
Thanks, Shabir!

The pull-request looks good to me. Resolving this thread.